presto-main/src/test/java/com/facebook/presto/sql/planner/assertions/WindowMatcher.java

```diff
@@ -46,24 +46,23 @@ public boolean matches(PlanNode node, Session session, Metadata metadata, Symbol
         }
 
         WindowNode windowNode = (WindowNode) node;
-        Collection<FunctionCall> actualCalls = windowNode.getWindowFunctions().values();
+        List<FunctionCall> actualCalls = windowNode.getWindowFunctions().values().stream().map(function -> function.getFunctionCall()).collect(toList());
 
         if (actualCalls.size() != functionCalls.size()) {
             return false;
         }
 
         LinkedList<FunctionCall> expectedCalls = new LinkedList<>(functionCalls);
```

Comment:
I think it was a leftover from an earlier implementation or WIP that didn't get caught before it got merged. As it's written now is correct to the intention: actualCalls is a linked list for removing stuff, and expectedCalls is used as a collection.